chpl/chpl-service/src/main/java/gov/healthit/chpl/entity/TestToolEntity.java

```diff
@@ -1,23 +1,30 @@
 package gov.healthit.chpl.entity;
 
 import java.util.Date;
+import java.util.List;
 
 import javax.persistence.Basic;
 import javax.persistence.Column;
 import javax.persistence.Entity;
+import javax.persistence.FetchType;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToMany;
 import javax.persistence.Table;
 
+import gov.healthit.chpl.domain.CertificationResultTestTool;
+
 
 @Entity
 @Table(name = "test_tool")
 public class TestToolEntity {
 	
 	@Id 
 	@GeneratedValue(strategy = GenerationType.IDENTITY)
-	@Basic( optional = false )
+	@Basic(optional = false)
+	@JoinColumn(name = "test_tool_id")
```

Comment:
Having @JoinColumn on a different ID broke something before when we were troubleshooting together... but this is working?